packages/breadcrumbs/src/elements/Breadcrumb.tsx

```diff
@@ -37,18 +40,29 @@ const renderItems = (items, getCurrentPageProps) => {
   });
 };
 
+export interface IBreadCrumbProps extends React.HTMLProps<HTMLDivElement> {
+  children?: React.ReactNode;
+}
+
 /** @component */
-export default function Breadcrumb({ children, ...breadcrumbProps }) {
+const Breadcrumb: React.FunctionComponent<IBreadCrumbProps> = ({
+  children,
+  ...breadcrumbProps
+}) => {
   const { getContainerProps, getCurrentPageProps } = useBreadcrumb();
 
   return (
-    /* role not needed as `BreadcrumbView` is a navigation landmark. */
-    <BreadcrumbView {...getContainerProps({ role: null })}>
-      <List {...breadcrumbProps}>{renderItems(children, getCurrentPageProps)}</List>
-    </BreadcrumbView>
+    /* role not needed as `StyledBreadcrumb` is a navigation landmark. */
+    <StyledBreadcrumb {...getContainerProps({ role: null } as any)}>
```

Comment:
I believe we removed the need to apply `{role: null}` in your recent a11y sweep of react-containers. If this is a `nav` element it may no longer be needed.

Reply:
I believe this is still needed. The recent [sweep](https://github.com/zendeskgarden/react-containers/pull/129) 🧹was regarding `aria-orientation` for `react-selection`.

We may need to do an additional sweep. Maybe we can sync on this after this PR.